src/odemis/driver/piezomotor.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, name, role, port, axes, inverted=None, param_file=None, **kwa
 
             Axis parameters:
                 axis_number (0 <= int <= 127): typically 1-3 for x-z, required
-                closed_loop (bool): True for closed loop (with encoder), default to True
+                closed_loop (bool): True for closed loop (with encoder), default to False
```

Comment:
I can't comment on unchanged lines, but I think the logging on line 136 is also written as if closed loop is the default. Even though on line 135 closed loop is set to false.

Reply:
Good catch!